outsystems/debugging.md

```diff
@@ -0,0 +1,11 @@
+# Debugging in live
+
+To debug in live follow these steps:
+* Switch the running mode of the production to development
+* In each of modules you wish to run the debugger in switch on debug mode
+* Run a solution to republish your application to allow you to debug
+* Set up the tenant you want to run on this url: http://e1aq2-pr32l0.outsystemsenterprise.com/
```

Comment:
Avoid publishing the url of the environments publicly on GitHub (maybe explain where to find the url of one specific server or other within Service Center instead?)